# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/wgpu-core/src/device/mod.rs
# Commit: b3109ad14685
# Full Hash: b3109ad14685624ada7861083ff5332aec9a7818
# Author: Andy Leiserson <aleiserson@mozilla.com>
# Date: 2025-10-07 17:00 +0000
# Description:
#   Bug 1992842 - Update wgpu to upstream 1072b878 (2025-10-06) r=webgpu-reviewers,supply-chain-reviewers,ErichDonGubler
# ==============================================================================

diff -r 1d3f000cf1f9 -r b3109ad14685 third_party/rust/wgpu-core/src/device/mod.rs
--- a/third_party/rust/wgpu-core/src/device/mod.rs	Tue Oct 07 16:51:44 2025 +0000
+++ b/third_party/rust/wgpu-core/src/device/mod.rs	Tue Oct 07 17:00:40 2025 +0000
@@ -35,10 +35,6 @@
 // value is enough for a 16k texture with float4 format.
 pub(crate) const ZERO_BUFFER_SIZE: BufferAddress = 512 << 10;
 
-// If a submission is not completed within this time, we go off into UB land.
-// See https://github.com/gfx-rs/wgpu/issues/4589. 60s to reduce the chances of this.
-const CLEANUP_WAIT_MS: u32 = 60000;
-
 pub(crate) const ENTRYPOINT_FAILURE_ERROR: &str = "The given EntryPoint is Invalid";
 
 pub type DeviceDescriptor<'a> = wgt::DeviceDescriptor<Label<'a>>;